be/src/olap/olap_define.h

```diff
@@ -189,11 +189,12 @@ enum OLAPStatus {
     // StorageEngine
     // [-500, -600)
     OLAP_ERR_ENGINE_INSERT_EXISTS_TABLE = -500,
-    OLAP_ERR_ENGINE_DROP_NOEXISTS_TABLE = -501,
-    OLAP_ERR_ENGINE_LOAD_INDEX_TABLE_ERROR = -502,
-    OLAP_ERR_TABLE_INSERT_DUPLICATION_ERROR = -503,
-    OLAP_ERR_DELETE_VERSION_ERROR = -504,
-    OLAP_ERR_GC_SCAN_PATH_ERROR = -505,
+    OLAP_ERR_ENGINE_INSERT_OLD_TABLET = -501,
```

Comment:
Better to append the new code as -506 and not change the exist codes to keep compatiable, assume some log analysis tool may depends on these code.